packages/terra-list/src/MultiSelectList.jsx

```diff
@@ -163,16 +163,10 @@ class MultiSelectList extends React.Component {
       newProps.isSelected = isSelected;
     }
 
```

Comment:
We can also update the wrap functions to remove isSelectable from this check:
if (item.props.isSelectable && this.shouldHandleSelection(index)) {